.vib/kapacitor/goss/goss.yaml

```diff
@@ -0,0 +1,9 @@
+gossfile:
+  # Load scripts from .vib/common/goss/templates
+  ../../common/goss/templates/check-binaries.yaml: {}
```

Comment:
I think you can add the `check-app-version` test as well:

```bash
$ kapacitor version
Kapacitor OSS 1.6.5 (git: HEAD c6c917f3097573544574ae94b5ef955a15256772)
```

Reply:
Weird, I thought I added it 😕 You are right though, thanks!